src/docs/Foundations/Layout/Grid/design.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: 'Design'
+order: 3
+status:
+  sketch: 'stable'
+  scss: 'stable'
+---
+
+> These are the recommendations to set up the values to use as your layout settings in your Sketch files.
+
+# Layout behaviors
+
+Grids on breakpoints from `S` to `M` are fluid.
+
+![fluid-grid](fluid-grid.png)
```

Comment:
The width and height properties does not work ? The one @nherwegh gave to you ? 
We need to use smaller size images

Reply:
I tried different variations of this code, It didn't work :/